### ISSUE TITLE
No '--gb' option in opts.py

### ISSUE THREAD

**Issue by utsavgarg** [yjxiong/tsn-pytorch#1]
All training commands have the parameter `--gb xx` but there is no such option in `opts.py`.

**Comment by bityangke** [comment]
`parser.add_argument('--clip-gradient', '--gd', default=None, type=float,
                    metavar='W', help='gradient norm clipping (default: disabled)')`

**Comment by bityangke** [comment]
@utsavgarg

**Comment by yjxiong** [comment]
Good question. This should be `--gd`. It stands for gradient clipping.

**Comment by yjxiong** [comment]
README updated. Thanks! @utsavgarg
